msystems/tests/saml_user_service_tests.py

```diff
@@ -25,10 +26,14 @@ def test_login(self):
         self.service.login(username=example_username,
                            user_data=example_user_data)
 
+        user_qs = InteractiveUser.objects.filter(
+            login_name=example_username, validity_to__isnull=True)
+
         self.assertTrue(User.objects.filter(
             username=example_username).exists())
-        self.assertTrue(InteractiveUser.objects.filter(
-            login_name=example_username, validity_to__isnull=True).exists())
+        self.assertTrue(user_qs.exists())
+        self.assertEquals(UserRole.objects.filter(user=user_qs.first()).first().role,
+                          Role.objects.filter(is_system=MsystemsConfig.EMPLOYER_ID).first())
```

Comment:
Please do not modify existing tests as it should all pass after your change. 
You should add new tests checking for role actions specifically.

Reply:
done